docs/cookbook/payments/custom-field-on-refund-payment.rst

```diff
@@ -0,0 +1,607 @@
+How to customize the refund form?
+=================================
+
+    .. note::
+        This cookbook describes customization of a feature available only with `Sylius/RefundPlugin <https://github.com/Sylius/RefundPlugin/>`_ installed.
+
+        A refund form is the form in which, as an Administrator, you can specify the exact amounts of money that will be refunded to a Customer.
+
+Why would you customize the refund form?
+----------------------------------------
+
+Refund Plugin provides a generic solution for refunding orders, it is enough for a basic refund but many shops need more custom functionalities.
+For example, one may need to add refund payments scheduling, as they may be paid once a month.
+
+How to add a field to the refund form?
+--------------------------------------
+
+The refund form is a form used to create the Refund Payment, thus in order to add a field to this form,
+you need to first add it to the Refund Payment's model.
+
+Refunds are processed with such a flow: ``command -> handler -> event -> listener``, and this flow we will also need to customize in order to process the data from the new field.
+
+In this customization, we will be extending the refund form with a ``scheduledAt`` field,
+which might be used then for scheduling the payments in the payment gateway.
+
+1. Add the custom field to the Refund Payment:
```

Comment:
```suggestion
**1. Add the custom field to the Refund Payment:**
```

You may consider adding **bold** on the enumeration, I'm not sure what was the syntax in sphinx-doc.